services/src/main/java/org/fao/geonet/services/schema/Info.java

```diff
@@ -182,10 +182,20 @@ public static Element getHelp(SchemaManager scm, Element elem, String fileName,
         if (result == null) {
         	result = checkEntries(scm, schema, entries, parent, name, isoType, false);
         }
+
+        // if we're in a schema that supports profiles (eg. iso19139, iso19115-2 or iso19115-3), then check the base schema
```

Comment:
Maybe we could even simplify this by checking if there is a "." in the schema identifier as this is general rule and then check the first part exists ?